utiliti/src/de/gurkenlabs/utiliti/Program.java

```diff
@@ -17,40 +17,54 @@
 
 public class Program {
   public static void main(String[] args) {
-    // setup basic settings
-    Game.info().setName("utiLITI");
-    Game.info().setSubTitle("LITIENGINE Creation Kit");
-    Game.info().setVersion("v0.5.1-beta");
-    Resources.strings().setEncoding(StandardCharsets.UTF_8);
-
-    // hook up configuration and initialize the game
-    Game.config().add(Editor.preferences());
-
-    Game.config().load();
     SwingUtilities.invokeLater(() -> {
-      UI.initLookAndFeel();
-      Game.init(args);
-      forceBasicEditorConfiguration();
-      Game.world().camera().onZoom(event -> Editor.preferences().setZoom((float) event.getZoom()));
-
-      // the editor should never crash, even if an exception occurs
       Game.setUncaughtExceptionHandler(new DefaultUncaughtExceptionHandler(false));
-
-      // prepare UI and start the game
-      UI.init();
-      Game.start();
-
-      // configure input settings
-      Input.mouse().setGrabMouse(false);
-      Input.keyboard().consumeAlt(true);
-
-      // load up previously opened project file or the one that is specified in
-      // the command line arguments
-      handleArgs(args);
-
-      String gameFile = Editor.preferences().getLastGameFile();
-      if (!Editor.instance().fileLoaded() && gameFile != null && !gameFile.isEmpty()) {
-        Editor.instance().load(new File(gameFile.trim()), false);
+      try {
+        try {
+          // setup basic settings
+          Game.info().setName("utiLITI");
+          Game.info().setSubTitle("LITIENGINE Creation Kit");
+          Game.info().setVersion("v0.5.1-beta");
+          Resources.strings().setEncoding(StandardCharsets.UTF_8);
+
+          // hook up configuration and initialize the game
+          Game.config().add(Editor.preferences());
+
+          Game.config().load();
+          
+          UI.initLookAndFeel();
+          Game.init(args);
+          forceBasicEditorConfiguration();
+          Game.world().camera().onZoom(event -> Editor.preferences().setZoom((float) event.getZoom()));
+    
+          // prepare UI and start the game
+          UI.init();
+          Game.start();
+        }
+        catch(Throwable t) {
+          throw new UtilitiInitializationError("Utiliti failed to initialize, see the stacktrace below for more information", t);

```

Comment:
Keep the correct capitalization in mind:  **utiLITI**

Reply:
Not really sure what to do with the class name, let me know if ab3fe0f is correct.

Reply:
works for me :) Just referencing the console output previously.